client/src/components/questions/QuestionList.jsx

```diff
@@ -0,0 +1,17 @@
+import React, {useState, useEffect} from 'react';
+import Question from './Question.jsx';
+import api from '../../lib/api.js';
+import StyleLib from '../../assets/Stylesheet.jsx';
+
+const QuestionList = ({displayedResults}) => {
+  const render = (result, index) => {
+    return <Question result={result} key={index}/>
+  }
+  return (
+    <section>
+      {displayedResults.map((result, index) => render(result, index))}
```

Comment:
`{displayedResults.map(render}`
You don't need to explicitly pass in the arguments.

Reply:
also will mention that this applies to lots of your components, I just happened to look at the imports for this one.